net/ftp-proxy/src/opnsense/mvc/app/views/OPNsense/FtpProxy/index.volt

```diff
@@ -103,7 +103,7 @@ POSSIBILITY OF SUCH DAMAGE.
                 <td></td>
                 <td>
                     <button data-action="add" type="button" class="btn btn-xs btn-default"><span class="fa fa-plus"></span></button>
-                    <button data-action="deleteSelected" type="button" class="btn btn-xs btn-default"><span class="fa fa-trash-o"></span></button>
+                    <!-- <button data-action="deleteSelected" type="button" class="btn btn-xs btn-default"><span class="fa fa-trash-o"></span></button> -->
```

Comment:
commented code is probably easier deleted than kept. we can always bring it back from the code repo itself.